crates/nu-command/src/system/run_external.rs

```diff
@@ -722,10 +710,6 @@ mod test {
             assert_eq!(actual, expected);
 
             let actual = expand_glob("./*.txt", cwd, Span::unknown(), &None).unwrap();
-            let expected = vec![
```

Comment:
I wrote this test intentionally - it should include the `./`, because if the user specified that in their glob, they probably expect it in the output.